content/exchange/artifacts/Windows.Detection.DensityScout.yaml

```diff
@@ -0,0 +1,37 @@
+name: Windows.Detection.DensityScout
+description: |
+  Finding malware on a potentially infected system by calculating the density of each file.
+  By default, it select PE (Portable Executable) files by checking them against the magic number ("MZ") 
+  of portable executables. This approach also matches portable executables with extensions one wouldn't expect.
+  Density under 0.1 is a fairly good threshold to start to think for suspicious files
+
+author: Raul Ramirez Gomez -- @rauramgom
+tools:
+  - name: densityscout
+    url: https://cert.at/media/files/downloads/software/densityscout/files/densityscout_build_45_windows.zip
```

Comment:
this is a zip file - you need to unzip it first before you can use it

another option is to host just the plain binary somewhere - i could not find the source code for this tool. Is it open source?